backend/curriculum_tracking/serializers.py

```diff
@@ -707,3 +686,18 @@ def save(self,**kwargs):
         instance.set_flavours(flavour_names)
         return instance
 
+
+
+class CourseRegistrationSerialiser(serializers.ModelSerializer):
+    class Meta:
+        model = models.CourseRegistration
+        fields = ["id", "user", "curriculum"]
```

Comment:
`user` always exposes the user id, that is the standard behavior. If you want to expose the user email then the field should be named `user_email` and the method should be called `get_user_email`. 

If there is a sensible default behavior then it's best not to override it.  You can add extra stuff. But arbitrarily changing the definition of a word is probably going to confuse someone. 

Same for the `curriculum` field.

Reply:
If Sam needs you to also expose the email address and curriculum name then that's fine, but those should be separate fields that are explicitly named.